package androidx.media3.decoder;

import androidx.annotation.Nullable;

/* loaded from: classes2.dex */
public class DecoderException extends Exception {
    public DecoderException(String str) {
        super(str);
    }

    public DecoderException(String str, @Nullable Throwable th2) {
        super(str, th2);
    }

    public DecoderException(@Nullable Throwable th2) {
        super(th2);
    }
}
